releasenotes/notes/0.5/0_5_release-89f59845afb19e89.yaml

```diff
@@ -0,0 +1,13 @@
+---
+prelude: >
+    The Qiskit Experiments 0.5 release added the visualization module for
+    drawing figures which replaces the previous plotting functionality. The
+    speed of randomized benchmarking experiments has been significantly
+    improved. The ``qubit`` and ``qubits`` input to experiments has been
+    regularized to ``physical_qubits``, and ``qiskit-ibmq-provider`` has been
+    deprecated in favor of ``qiskit-ibm-provider``. A new
```

Comment:
I am wondering what the user should take away from the provider change. qiskit-ibm-provider is not a hard requirement now. Maybe "Support for qiskit-ibmq-provider has been deprecated in favor of support for qiskit-ibm-provider"?

Reply:
Done.